plugins/pulp_rpm/plugins/importers/yum/depsolve.py

```diff
@@ -72,9 +72,6 @@ def __cmp__(self, other):
                       a positive value if self is greater than other.
         :rtype:       int
         """
-        if self.name != other.name:
```

Comment:
Can you explain why this is removed? Comparing ``kernel-4.5.7-200`` and ``kernel-devel-4.5.7-200`` doesn't make any sense, and hearing that they're actually equal doesn't seem like the right behaviour.